templates/component-mapper/src/src/files/checkbox.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const MultipleChoiceList = () => 'MultipleChoiceList';
```

Comment:
I would definitely prepare example using the common multiple-choice component from common package